.github/workflows/pypi.yml

```diff
@@ -25,10 +27,21 @@ jobs:
         python -m pip install --upgrade pip
         pip install setuptools wheel twine
 
-    - name: Build and publish
+    - name: Build
+      run: |
+        python setup.py sdist bdist_wheel
+
+    - name: Publish
+      if: ${{ 'release' == github.event_name }}
       env:
         TWINE_USERNAME: __token__
         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python setup.py sdist bdist_wheel
+      run:
         twine upload dist/*
+
+    - name: Test for creation sphinx documentations
+      if: ${{ 'release' != github.event_name }}
+      run: |
+        cd docs
+        pip install -r requirements.txt
+        make html
```

Comment:
Do we have a negative example for this test? What will happen if `sphinx` returns error?

Reply:
In previous PR i added default key -W to fail in warning case